Source/RxSwift/mergeWith.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  mergeWith.swift
+//  RxSwiftExt
+//
+//  Created by Joan Disho on 12.05.18.
+//  Copyright © 2018 RxSwiftCommunity. All rights reserved.
+//
+
+import Foundation
+import RxSwift
+
+extension Observable {
+    /**
+     Merges `self` with another observable.
+
+     - parameter other: An other observable.
```

Comment:
parameter should be `with` since that's the name the user actually sees, and not the internal one